### PR TITLE
Localizables in dependant projects

### PR DESCRIPTION
**Pull request by mmiedlarz**
Looks like there was a missing entry for a resources in our podspec.
Also I changed implementation of our `.localizable` extension to distinguish different bundles in dependant projects.

### CODE REVIEW

**Review by DaDer — APPROVED** [review]
Code: 👍

File: CreatubblesAPIClient.podspec [hunk 1]
```diff
@@ -91,7 +91,7 @@ Pod::Spec.new do |s|
   #
 
   # s.resource  = "icon.png"
-  # s.resources = "Resources/*.png"
+  s.resources = "CreatubblesAPIClient/Resources/Localizations/**/*.strings"
```
Comment (DaDer) [inline CreatubblesAPIClient.podspec:94]: 👍

**Review by sebastianewak — APPROVED** [review]
Code: +1
Func: n\a

**Review by JakubJankowski — APPROVED** [review]
👍

**Pull request merged**